### ISSUE TITLE
Totally lost

### ISSUE THREAD

**Issue by VitruviusX** [exercism/exercism#4943]
So I was trying to use Exercism/CLI? and I already have python 3.7 installed. I manually install Exercism to windows fine.  In order to run the tests it says to install pytest but to install pytest I need pip. But I think that python 3.7 already has pip. But I still need to install pytest. Somehow I ended up in the command line for my computer. command line didn't recognize python or pip. It didn't recognize python I think because the PATH and environmental and local variables were wrong? But I can only change the PATH temporarily through the command line and I need access to the system settings to change PATH to include python so that I can use command line to install pytest so I can use Exercism/CLI? Nothing I typed into command line was recognized except one thing I copied and pasted about setx and python37 added something to the directory, I cant look at it again because it said to reboot command line to have it do something, python still wasn't recognized in there. 

This seems awfully confusing/complicated to do the first exercise which is Hello_World! on the python path. I don't know much about computers so this is a tough line to start on. 

tldr; installed exercism/CLI, cant get pytest, command line wont recognize python, PATH, environmental variables, setx did something. 

Zac
